pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -158,8 +158,7 @@ class TestCasting(base.BaseCastingTests):
 
 
 class TestGroupby(base.BaseGroupbyTests):
-    def test_groupby_agg_extension(self, data_for_grouping):
-        super().test_groupby_agg_extension(data_for_grouping)
+    pass
```

Comment:
I changed this to `pass` because the classes directly above and below both had `pass`.  Is that the right thing to do?

Reply:
Yup. Thanks